playbooks/roles/mailing-list/templates/postfix-main.cf

```diff
@@ -0,0 +1,29 @@
+# File: main.conf
+
+# Plebia relays incomming email for help@opencraft.com this host
+
+mydomain = opencraft.com
+myhostname = mailinglist.opencraft.com
+
+# send email as <user>@opencraft.com
+myorigin = $mydomain
+
+# Only accept email destined for opencraft.com or mailinglist.opencraft.com
+mydestination = $mydomain, $myhostname
+
+# Only relay mail from the local mailman server
+mynetworks_style = host
+
+# Support the default VERP delimiter.
+recipient_delimiter = +
+
+# Don't treat owner- and -request addresses specially
+owner_request_special = no
+
+# Let mailman3 manage the transport and relay
```

Comment:
@tartansandal I thought postfix was supposed to do the transport part?

Reply:
>  thought postfix was supposed to do the transport part?

@swalladge Yup that is exactly what we are doing.  What the above setting does is allow `mailman` to manage how `postfix` handles the transport (by changing its configuration). This way you can add/modify mailing lists in mailman without having to explicitly touch the postfix transport configuration. Make sense?

Reply:
Yep, cool :+1: